### DISCUSSION TITLE
Welcome to volley34-mobile Discussions!

### DISCUSSION THREAD

**Discussion by eric-taix** [eric-taix/volley34-mobile#199] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eric-taix** [comment]
@Lo0ping @kverbard @matt-aubert This is the space where we can talk about ideas instead of using issues ;-)

**Comment by Lo0ping** [comment]
Hi, 
Somebody can explains me Farce Statistics ?
And why values are from 0 to 200% instead of 0 to 100% (that will be more logical) ?

**Reply by eric-taix** [reply]
Yes of course. 
 - Force is not an absolute value! It is computed relatively to other competitors (in the same pool)
 - The attack force is: `the average points per set won by the team / the average points per set won by all the teams`
 - The defense force is: `(25 - the average points per set conceded by the team) / (25 - the average points per set conceded by all the teams)`

So if a force is at the middle, it means that the team has the same force as the average force of teams in this pool. A higher value means that the team is better than the average teams, etc...

Note: This formula has been extracted from an excellent article about computing forces in sport (it's not an invention on my part)

As these values are relative to other teams, the range is not limited to 0% - 200% but to 0% - Infinity. In the application I have visually limited the values to 75% - 125%.

But I'm wondering how did you say that values are limited to 0% - 200%? We do not display values anymore to avoid this kind of question.

**Reply by Lo0ping** [reply]
I have see it on a previous version, I think, with VLM /1 Force around 190%.So I have estimate that the range was 0..200

**Comment by Lo0ping** [comment]
Don't you think it would be a good idea to have a meeting (face-to-face or remote) to speak about the project ?

**Reply by eric-taix** [reply]
Yes I think can be a very good idea 👍
